File: Modules/src/org/oemap/services/beans/settings/SettingBase.java
/*******************************************************************************
 * Written 2013 by Per Fahlén, Sweco Postion
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the GNU Affero General Public License
 * which accompanies this distribution, and is available at
 * http://www.gnu.org/licenses/agpl-3.0.html
 ******************************************************************************/

package org.oemap.services.beans.settings;

import se.unlogic.standardutils.dao.annotations.DAOManaged;
import se.unlogic.standardutils.xml.GeneratedElementable;
import se.unlogic.standardutils.xml.XMLElement;

public class SettingBase extends GeneratedElementable {

	@XMLElement
	@DAOManaged
	private String name;

	@XMLElement
	@DAOManaged
	private String url;

	@XMLElement
	@DAOManaged
	private String username;

	@XMLElement
	@DAOManaged
	private String password;

	@XMLElement
	@DAOManaged
	private String note;

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getUrl() {
		return url;
	}

	public void setUrl(String url) {
		this.url = url;
	}

	public String getUsername() {
		return username;
	}

	public void setUsername(String username) {
		this.username = username;
	}

	public String getPassword() {
		return password;
	}

	public void setPassword(String password) {
		this.password = password;
	}

	public String getNote() {
		return note;
	}

	public void setNote(String note) {
		this.note = note;
	}

}

File: UI/src/main/javascript/model/SettingBase.js
Ext.define('AdmClient.model.SettingBase', {
	extend : 'Ext.data.Model',
	fields : [  {
		name : 'name',
		type : 'string'
	}, {
		name : 'url',
		type : 'string'
	}, {
		name : 'username',
		type : 'string'
	}, {
		name : 'password',
		type : 'string'
	}, {
		name : 'note',
		type : 'string'
	},{
		name : 'ID',
		type : 'integer'
	}]
});